tests/test_area.py

```diff
@@ -55,6 +55,7 @@ def setUp(self):
         self.config.ticks_per_slot = int(self.config.slot_length.seconds /
                                          self.config.tick_length.seconds)
         self.config.start_date = today(tz=TIME_ZONE)
+        GlobalConfig.sim_duration = duration(days=1)
```

Comment:
Why is this needed?, you have already refactored the below usages to use `self.config`

Reply:
self.config changes only the SimulationConfig instance unfortunately. GlobalConfig is a different object, which includes global default settings for the configuration parameters. This uses class variables deliberately in order to be a singleton-style object, with the typical singleton disadvantage: if the class member is mutated, then the mutation affects all instances of this class. That's why changes in this object persist throughout multiple test runs.